### DISCUSSION TITLE
Fly.io integration

### DISCUSSION THREAD

**Discussion by Wavelent** [FuseFairy/DiscordBot-EdgeGPT#21] [Q&A]
Hello, i dont know if a problem with me or what, but i can use everything localy but when i try to deploy bot on fly.io i get this. Tried everything but nothing work, 
![image](https://user-images.githubusercontent.com/80627564/226453053-5d530037-6e81-4a36-a8ee-6787a2f9de6f.png)

**Comment by FuseFairy** [comment]
Although you seem to have solved the problem, I would still like to guess what the problem is. When you build the flyio app, you chose to automatically generate the .dockerignore file, which will be directly based on the .gitignore file, but there is an .env file in .gitignore, so .dockerignore will also ignore the .env file, and finally the .env will not be uploaded to the server.
